drivers/overlay/overlay.go

```
@@ -595,11 +624,12 @@ func (d *Driver) create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 	}
 	if parent != "" {
 		st, err := system.Stat(d.dir(parent))
-		if err != nil {
+		if err == nil {
+			rootUID = int(st.UID())
+			rootGID = int(st.GID())
+		} else if pLayer := d.additionalLayer(parent); pLayer == "" {
 			return err
-		}
-		rootUID = int(st.UID())
-		rootGID = int(st.GID())
+		} // TODO: rootUID & rootGID of additional layer?
```

You are ignoring the error, Should you at least log it?